zcash_test_vectors/zip_0244_zsa.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import sys; assert sys.version_info[0] >= 3, "Python 3 required."
+
+from hashlib import blake2b
+import struct
+
+from .transaction_zsa import (
+    TransactionZSA, IssueActionDescription,
+)
+from .output import render_args, render_tv
+from .rand import Rand
+from .zip_0143 import (
+    SIGHASH_ALL,
+    SIGHASH_ANYONECANPAY,
+    SIGHASH_NONE,
+    SIGHASH_SINGLE,
+)
+
+# - Burn
+def orchard_zsa_burn_digest(tx: TransactionZSA):
```

Comment:
`tx: TransactionZSA`
 if you use it here, make sure all functions in this file use it as well. (all new code) (consistency)

Reply:
two new lines before a function, as discussed

Reply:
Done